traitsui/qt4/constants.py

```diff
@@ -45,7 +45,12 @@
 
 # Screen size values:
 
-_geom = QtGui.QApplication.desktop().availableGeometry()
+# QDesktopWidget.availableGeometry(int screen) is deprecated and Qt docs
+# suggest using screens() instead, but screens in not available in qt4
+if QtCore.__version_info__ >= (5, 0):
```

Comment:
`__version_info__` from Qt is actually a tuple with three values. While we can do `>` and `<` comparison with a smaller tuple, `==` does not work that way. The safest thing to do here might just be to take the first element of `__version_info__` so that comparison is not sensitive to the length of `__version_info__`.
(Python `sys.version_info` is a tuple with 5 entries.)

Reply:
Oh no, taking it back. This is okay.